### DISCUSSION TITLE
requirements.txt file

### DISCUSSION THREAD

**Discussion by SHEYCHAE** [kikiru328/Bone_Detection#59] [CHAT]
[requirements.txt](https://github.com/kikiru328/Bone_Detection/files/7492180/requirements.txt)

한번 새로운 가상환경 만든 후에  (가상환경 경로에) 다운받아서
새로운 가상환경 activate 해서 install 해봐주시고 
(pip install -r requirements.txt )

pip list로 제대로 들어가는지 확인해주세요

**Comment by Jjongu** [comment]
제 컴에서 새 가상환경을 만들어도 이미 라이브러리가 다 설치되어 있어서 차이가 없습니다

**Reply by kikiru328** [reply]
신촌에 와서 보여주세요

**Comment by SHEYCHAE** [comment]
matplotlib
sklearn추가됨 
[requirements.txt](https://github.com/kikiru328/Bone_Detection/files/7492290/requirements.txt)

**Reply by SHEYCHAE** [reply]
requirements.txt install할 때

- cmd 관리자 모드로 실행
- pip install --user -r requirements.txt 
(다운받는데 3분정도 걸림)

**Reply by hye-jj** [reply]
감사합니다~!!

**Comment by SHEYCHAE** [comment]
[requirements.txt](https://github.com/kikiru328/Bone_Detection/files/7496158/requirements.txt)
진짜 찐막 requirements.txt    211108

**Comment by SHEYCHAE** [comment]
[requirements.txt](https://github.com/kikiru328/Bone_Detection/files/7501081/requirements.txt)
REAL 개찐막.

**Comment by SHEYCHAE** [comment]
[requirements.txt](https://github.com/kikiru328/Bone_Detection/files/7501097/requirements.txt)
THIS IS TOTALLY LAST ONE. 
REA~~~~~~~~~~~~~~~~~~~~~L 마 지 막

**Comment by SHEYCHAE** [comment]
[requirements.txt](https://github.com/kikiru328/Bone_Detection/files/7508906/requirements.txt)
pyqt5 추가됨

**Comment by hye-jj** [comment]
최종!!!!!
[requirements.txt](https://github.com/kikiru328/Bone_Detection/files/7509280/requirements.txt)
!

**Comment by SHEYCHAE** [comment]
[requirements.txt](https://github.com/kikiru328/Bone_Detection/files/7509300/requirements.txt)

이것도 win32 되는거!@ 둘중 하나 다운받아 쓰면 됨

**Comment by hye-jj** [comment]
@SeunghyeChae  가 찾은 방법!
pip freeze > requirements.txt # 패키지를 requirements.txt로 저장

pip install -r requirements.txt # 해당 패키지를 설치
